description: try alternative locations for libwidevinecdm.so
  - $HOME/.local/lib/ (snap-friendly, see https://launchpad.net/bugs/1738149)
author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/chrome/common/chrome_paths.cc
+++ b/chrome/common/chrome_paths.cc
@@ -338,6 +338,11 @@ bool PathProvider(int key, base::FilePat
 #if (defined(OS_LINUX) || defined(OS_CHROMEOS)) && \
     BUILDFLAG(BUNDLE_WIDEVINE_CDM)
     case chrome::DIR_BUNDLED_WIDEVINE_CDM:
+      base::PathService::Get(base::DIR_HOME, &cur);
+      cur = cur.Append(FILE_PATH_LITERAL(".local/lib/libwidevinecdm.so"));
+      if (base::PathExists(cur)) {
+        break;
+      }
       if (!GetComponentDirectory(&cur))
         return false;
 #if !defined(OS_CHROMEOS)
